keyboards/system76/system76_ec.c

```diff
@@ -172,6 +174,16 @@ void raw_hid_receive(uint8_t *data, uint8_t length) {
                 }
             }
             break;
+        case CMD_LED_GET_MODE:
+            data[2] = rgb_matrix_get_mode();
```

Comment:
I blocked get when unlocked too, since it did not make much sense to read LED information at that time